app/build.gradle

```diff
@@ -115,6 +126,8 @@ dependencies {
     // Dagger
     implementation "com.google.dagger:dagger:$dagger_version"
     implementation "com.google.dagger:dagger-android-support:$dagger_version"
+    implementation 'com.android.support.constraint:constraint-layout:1.1.3'
```

Comment:
When adding new dependencies please try to add an extension for every version ass dagger is using now

Reply:
What happened with this?